sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 ## 1.0.0-preview.3 (Unreleased)
 
 - Blank pages in receipt recognition are now handled properly.
+- Support Azure Active Directory credential.
+- Support to copy a custom model from one Form Recognizer resource to another.
+- Refactoring for cross-language consistency:
+  - [Breaking] Rename `beginRecognizeForms()` to `beginRecognizeCustomForms()` in `FormRecognizerClient`.
+  - [Breaking] Rename `listModels()` to `listCustomModels()` in `FormTrainingClient`.
+  - [Breaking] Rename `count` to `customModelCount` and `limit` to `customModelLimit` in `AccountProperties`.
+  - [Breaking] Rename type `ErrorInformation` to `FormRecognizerError`.
+  - [Breaking] Rename type `ModelStatus` to `CustomFormModelStatus`.
+  - [Breaking] Rename type `CustomFormSubModelField` to `CustomFormField`.
+  - [Breaking] Rename type `FormElement` to `FormContent` and `FormElementCommon` to `FormContentCommon`.
+  - [Breaking] Rename property `fieldLabel` to `labelText` in `FormField` type.
+  - [Breaking] Rename type `ModelInfo` to `CustomFormModelInfo`.
+  - [Breaking] Rename tppe `TrainStatus` to `TrainingStatus`.
+  - [Breaking] Rename `models` to `submodels` in `CustomFormModel`.
+  - [Breaking] Recognition methods and training methods now return the result directly, instead of wrapping them in a response object. Specifically,
+    - `beginTraining` now returns `CustomFormModel` instead of `FormModelResponse` from the poller.
+    - `beginRecognizeContent` and `beginRecognizeContentFromUrl` now return `FormPageArray` instead of `RecognizeContentResultResponse` from the poller.
+    - `beginRecognizeForms` and `beginRecognizeFormsFromUrl` now return `RecognizedFormArray` instead of `RecognizeFormResultResponse` from the poller.
+    - `beginRecognizeReceipts` and `beginRecognizeReceiptsFromUrl` now return `RecognizedReceiptArray` instead of `RecognizeReceiptResultResponse` from the poller.
+  - [Breaking] Remove `getFormTrainingClient()` from `FormRecognizerClient`. A new method `getFormRecognizerClient()` is added to `FormTrainingClient`
+  - Rename the first parameter of `beginRecognizeContentFromUrl()` from `documentUrl` to `formFileUrl`.
+  - Rename the second parameter of `beginRecognizeFormsFromUrl()` from `documentUrl` to `formFileUrl`.
+  - Rename the first parameter of `beginRecognizeReceiptsFromUrl()` from `documentUrl` to `receiptFileUrl`.
```

Comment:
we renamed the `*url` parameter to `formUrl` and `receiptUrl`

Reply:
I just did it in PR https://github.com/Azure/azure-sdk-for-js/pull/9118

Reply:
approved